src/main/java/mate/academy/model/User.java

```diff
@@ -33,4 +47,49 @@ public class User {
     private String lastName;
     @Column(name = "shipping_address")
     private String shippingAddress;
+    @ToStringExclude
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(
+            name = "users_roles",
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id")
+    )
+    private Set<Role> roles = new HashSet<>();
+    @Column(nullable = false)
```

Comment:
Did you have checkstyle in your project?

```suggestion
    private Set<Role> roles = new HashSet<>();
    
    @Column(nullable = false)
```

Reply:
Yes, I added checkstyle in the begining when I created the repository. Have you found any more checkstyle errors?

Reply:
Not yet, you can run mvn clean package to check

Reply:
I always run mvn clean package before push. I didn't notice because checkstyle didn't detect the absence of an empty string between roles and is_deleted as an error. Do I understand correctly that we should separate the collection-initialized field and the regular field?